engine/table/src/test/java/io/deephaven/engine/table/impl/updateby/TestUpdateByGeneral.java

```diff
@@ -5,32 +5,38 @@
 import io.deephaven.api.updateby.OperationControl;
 import io.deephaven.api.updateby.UpdateByOperation;
 import io.deephaven.engine.table.Table;
+import io.deephaven.engine.table.impl.QueryTable;
 import io.deephaven.engine.testutil.EvalNugget;
 import io.deephaven.engine.table.impl.TableDefaults;
 import io.deephaven.api.updateby.UpdateByControl;
+import io.deephaven.engine.testutil.TstUtils;
 import io.deephaven.engine.testutil.generator.TestDataGenerator;
 import io.deephaven.engine.testutil.generator.SortedDateTimeGenerator;
 import io.deephaven.engine.updategraph.UpdateGraphProcessor;
-import io.deephaven.test.types.ParallelTest;
+import io.deephaven.engine.util.TableTools;
+import io.deephaven.test.types.OutOfBandTest;
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
 
 import java.time.Duration;
 import java.util.*;
 
 import static io.deephaven.engine.testutil.GenerateTableUpdates.generateAppends;
+import static io.deephaven.engine.testutil.TstUtils.*;
 import static io.deephaven.engine.testutil.testcase.RefreshingTableTestCase.simulateShiftAwareStep;
-import static io.deephaven.engine.testutil.TstUtils.validate;
+import static io.deephaven.engine.util.TableTools.col;
+import static io.deephaven.engine.util.TableTools.intCol;
 import static io.deephaven.time.DateTimeUtils.MINUTE;
 import static io.deephaven.time.DateTimeUtils.convertDateTime;
 
-@Category(ParallelTest.class)
+@Category(OutOfBandTest.class)
 public class TestUpdateByGeneral extends BaseUpdateByTest {
 
     @Test
     public void testMixedAppendOnlyZeroKey() {
         for (int size = 10; size <= 10000; size *= 10) {
             for (int seed = 10; seed < 20; seed++) {
+                System.out.println("Stating test: size " + size + ", seed " + seed);
```

Comment:
I think it might be better to turn down this spam by default. Maybe guard under `if (RefreshingTableTestCase.printTableUpdates)`. Correct `Stating` to `Starting` if you're keeping these.

Reply:
This is part of the bug-hunt, will unspam this before merge